app/controllers/telegram_webhooks_controller.rb

```diff
@@ -97,4 +66,108 @@ def help(*)
   def action_missing(action)
     reply_with :message, text: "Can not perform #{action}" if command?
   end
-end
+
+  private
+  def getArgsFromCommand(*args)
+    #Day informed as a number
+    if args[0].match(NUMBER_REGULAR_EXPRESSION)
+
+      dayFromArgs = args[0].to_s.rjust(2, '0')
+      isCurrentMonth = true
+
+    elsif args[0].match(DATE_REGULAR_EXPRESSION)
+
+      dateSplitted = args[0].split('/')
+      dateFormated = Date.strptime(dateSplitted[0] + "/" + dateSplitted[1] + "/" + getCurrentYear, t(:date_format))
```

Comment:
@filipecosta01 I myself consider interpolation cleaner than concatenation. I'm not saying you have to change it here, but maybe you should take a look on how to do it so you can use it in the future.

Here we'd have something like: Date.strptime("#{dateSplitted[0]}/#{dateSplitted[1]}/#{getCurrentYear}", t(:date_format))